1_fetch/src/fetch_FORESCE.R

```diff
@@ -0,0 +1,27 @@
+download_backcasted_LC <- function(sb_id, filename, download_path, overwrite_file = T){
```

Comment:
Not that this is rather generic function and possible redundant to what is already in fetch. Adding point to consider combining `sb::item_file_download()` functions at a later point.